worker/uniter/runlistener.go

```diff
@@ -143,9 +143,9 @@ func (s *RunListener) Close() error {
 }
 
 // RunCommands executes the supplied commands in a hook context.
-func (r *RunListener) RunCommands(args RunCommandsArgs) (results *exec.ExecResponse, err error) {
+func (s *RunListener) RunCommands(args RunCommandsArgs) (results *exec.ExecResponse, err error) {
```

Comment:
Why change the receiver fro "r" to "s"?

Reply:
sure, to keep consistency, I will change them all to `r`